drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/dmn/KogitoScenarioSimulationBuilder.java

```diff
@@ -331,6 +336,29 @@ private Simulation getDMNSimulation(final FactModelTuple factModelTuple) {
         return toReturn;
     }
 
+    /**
+     * It defines the column width of a new column, based on its
+     * <code>FactMapping.expressionIdentifier.getName()</code> field
+     * @param expressionIdentifierName
+     * @return
+     */
+    protected double getColumnWidth(String expressionIdentifierName) {
```

Comment:
I remember a similar (the same?) code somewhere else. Am I wrong?

Reply:
@danielezonca Yes, in backend side

Reply:
Ok in this case it is fine for me because I would like to keep only client side code as soon as it will be stable enough and the backend one could be removed